### DISCUSSION TITLE
Change the WPS icon in the custom theme

### DISCUSSION THREAD

**Discussion by boris-gu** [OnionUI/Onion#1200] [Support] [answered]
Can I do this? If so, what path/name should the icon have?
The problem is that the original icon overlaps the battery icon. Moreover, this problem also exists in the default theme, these icons are too close to each other

![Tweaks_005](https://github.com/OnionUI/Onion/assets/44787162/8d40b40e-37f1-49a3-9304-8f44828f7675)

![Tweaks_012](https://github.com/OnionUI/Onion/assets/44787162/13f15395-53a9-4be7-a1f9-fcc7cfd33705)

**Comment by XK9274** [comment]
I'll rework the icon on this. It wasn't changed when the themes/WiFi icon changes were added

**Reply by boris-gu** [reply]
Will there be an option in the future to change this icon in custom themes?

**Reply by XK9274** [reply, marked as answer]
Sure, I can add that when it's reworked.
